config.nix

```diff
@@ -9,11 +9,13 @@
     keystore_file = "agent.key";
     public_address = "HcScIio76M8cbDqhrsZKp37W8YZ4y8X9v9epzbyhRzqpjeb5sigdC4puN6b5koi";
   }
-  {
-    id = "hp-admin-agent-2";
-    name = "HP Admin 2";
-    keystore_file = "agent-2.key";
-    public_address = "HcScjSPMA9g46x6nosEih9thqwvesx53udZEkqMfqcwnbw9sZKPSyIFNvtfxrqi";
-  }];
+  # Currently we can only run one agent on a conductor
+  #{
+  #  id = "hp-admin-agent-2";
+  #  name = "HP Admin 2";
+  #  keystore_file = "agent-2.key";
+  #  public_address = "HcScjSPMA9g46x6nosEih9thqwvesx53udZEkqMfqcwnbw9sZKPSyIFNvtfxrqi";
+  #}
```

Comment:
https://kentcdodds.com/blog/please-dont-commit-commented-out-code
https://stackoverflow.com/questions/3664288

Reply:
<3

Reply:
Reason: This was left in because when we spin up two conductors soon (This will be done soon. Hopefully this week). We would need this.

**But I am happy to remove the commented code out.**